io.github.chen08209.FlClash.yml

```diff
@@ -0,0 +1,44 @@
+id: io.github.chen08209.FlClash
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+separate-locales: false
+command: AppRun
+
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=session-bus
```

Comment:
this is not allowed, you should figure out which specific busnames are needed by running it with `flatpak run --log-seesion-bus --nosocket=session-bus $app_id`